version-mappings.yml

```diff
@@ -6,9 +6,10 @@ mappings:
     compatibleTestsVersions:
       - "202210.00"
       - "202210.01"
-  - freeRTOSVersion: "202211.00"
+      - "202210.02"
+  - freeRTOSVersion: "202212.00"
     compatibleTestsVersions:
-      - "202210.01"
+      - "202210.02"
```

Comment:
Why is 202210.00-LTS compatible with 202210.01 and 202210.00, yet the new release is only compatible with 202210.02. Shouldn't both 202210.00-LTS and 202212.00 be compatible with the same test versions?